admin/broadleaf-open-admin-platform/pom.xml

```diff
@@ -211,5 +211,33 @@
             <artifactId>spock-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Same as above. Also lets not define versions in these inner level poms. It is easy to maintain versions if all versions are in the top level pom. Line 688 in Broadleaf/pom.